index.html

```diff
@@ -0,0 +1,63 @@
+                </a>
+                <a href="https://www.twitter.com/dara_soji">
+                    <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAQhJREFUSEvtldERwUAURU8qoAN0oAM6QAdUgApQASqgBDqgAyUogQqYa3bN2tlkk4wYH7k/ycjuPfvue4mEipVU7E8NiCbsRjQGdsANWALb6O7PBdq/Nj/Ngb3uXcAVaDl7DsDEALNYTWMsgJW8Oj7gEXDRwhFwSSF0TdW6+nod3q0gBLCbZoHIbCSqIKRCABnYyHSvXg0jPSoMkJ8ik9o5BqAUIIfve0kUcAT6QCPiegdOwMBbFwWsgI3JvpcCOZteaAgWZQB64SRdfQMdIOt5rgqsgSCKS5MkaYIUi1XoAIUBMrMzr8+Jq68B0vr9HwBNzNQ5otvE2Pz7FehLrMmq/9Fiyf0goide8TgZZ2khYwAAAABJRU5ErkJggg==" alt="gmail logo"/>
+                </a>
+            </div>
+        </div>
+
+        <div class="background_styling">
+            <div class="ellipse_3">
+                <svg  viewBox="0 0 210 213" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M209.581 57.0278C200.949 41.7052 188.83 28.6308 174.205 18.863C159.581 9.09529 142.861 2.90884 125.401 0.804592C107.94 -1.29966 90.2299 0.737445 73.7033 6.75095C57.1767 12.7645 42.2984 22.5853 30.2733 35.4184C18.2481 48.2515 9.41404 63.736 4.48635 80.6183C-0.441345 97.5005 -1.32415 115.306 1.90941 132.593C5.14297 149.88 12.402 166.162 23.0988 180.122C33.7955 194.081 47.6293 205.326 63.4801 212.945L84.3582 169.508C75.3281 165.168 67.447 158.762 61.3531 150.809C55.2591 142.856 51.1237 133.58 49.2815 123.732C47.4394 113.883 47.9423 103.74 50.7496 94.1219C53.5569 84.5041 58.5896 75.6825 65.4404 68.3716C72.2911 61.0606 80.7672 55.4656 90.1824 52.0397C99.5976 48.6138 109.687 47.4533 119.634 48.6521C129.582 49.8509 139.107 53.3753 147.438 58.94C155.77 64.5046 162.674 71.9532 167.592 80.6824L209.581 57.0278Z" fill="#6070FF"/>
+                </svg>
+            </div>
+            <div class="ellipse_12">
+                <svg width="25" height="13" viewBox="0 0 25 13" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M0.496961 -0.000453654C0.278167 3.19352 1.33714 6.3436 3.44091 8.75679C5.54468 11.17 8.52093 12.6486 11.7149 12.8674C14.9089 13.0862 18.059 12.0272 20.4722 9.92347C22.8853 7.8197 24.364 4.84345 24.5828 1.64948L0.496961 -0.000453654Z" fill="#6070FF"/>
+                </svg>
+            </div>
+            <div class="rectangle_5">
+                <svg  viewBox="0 0 68 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <rect x="4.82422" y="-0.00012207" width="68.0631" height="12.1762" transform="rotate(23.34 4.82422 -0.00012207)" fill="#EBEBFF"/>
+                </svg>
+            </div>
+            <div class="ellipse_5">
+                <svg  viewBox="0 0 11 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <ellipse cx="5.30398" cy="6.21088" rx="5.5" ry="5" transform="rotate(52.9708 5.30398 6.21088)" fill="#CBCDFF"/>
+                </svg>
+            </div>
+        </div>
```

Comment:
- [ ] Regarding the background styles, kindly avoid using markup just for decorative images, as they serve just for visual purposes and don't provide meaningful content to your page. Instead, you could use the CSS's `background-image` property in your `Headline content` container to achieve the same! 🤓 🔧 

💡 _Hint: I noticed that you downloaded the SVG for each one of the elements, please avoid doing it for these kinds of compositions as laying them out would be very hard. Instead, FIGMA lets you download the entire background image, please take this approach!_